ts/features/fims/store/selectors/index.ts

```diff
@@ -0,0 +1,13 @@
+import { createSelector } from "reselect";
+import { GlobalState } from "../../../../store/reducers/types";
+
+const fimsFeatureSelector = (state: GlobalState) => state.features.fims;
```

Comment:
See comments below but this can be removed since we do not need to use `createSelector`